db/db_write_test.cc

```diff
@@ -68,7 +69,14 @@ TEST_P(DBWriteTest, IOErrorOnWALWritePropagateToWriteThreadFollower) {
     threads.push_back(port::Thread(
         [&](int index) {
           // All threads should fail.
-          ASSERT_FALSE(Put("key" + ToString(index), "value").ok());
+          auto res = Put("key" + ToString(index), "value");
+          if (!options.manual_wal_flush) {
```

Comment:
Maybe you can do it like this:
>>>
if (options.manual_wal_flush) {
    res = dbfull()->FlushWAL(false);
}
ASSERT_FALSE(res.ok());
>>>

It's cleaner.